### PR TITLE
Apply oracle handler to convert unicode input to byte strings within DBCore code; revert patch for RESTModel

### PR DESCRIPTION
**Pull request by vkuznet**

### CODE REVIEW

**Comment by ericvaandering** [comment]
Looks like this blows up MySQL

**Comment by vkuznet** [comment]
Now, once I see the error I'll fix it.

**Comment by ericvaandering** [comment]
Much better.

File: src/python/WMCore/Database/DBCore.py [hunk 1]
```diff
@@ -134,6 +147,8 @@ def processData(self, sqlstmt, binds={}, conn=None,
                 connection = self.connection()
             else:
                 connection = conn
+            if  isinstance(self.engine.dialect, OracleDialect_cx_oracle):
```
Comment (amaltaro) [inline src/python/WMCore/Database/DBCore.py]: Valentin, I know you like leaving two white spaces in an `if` clause, unfortunately it generates one more complaint line in the pylint output. You just told me you'll probably not have time today, thus please avoid them in future PRs.

**Comment by amaltaro** [comment]
@vkuznet @ericvaandering I was about to miss this one, I believe we do want to have it in for tomorrow's tag, don't we?

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
And before I forget, @vkuznet please squash these commits :)

**Comment by vkuznet** [comment]
Alan, I removed white space and squashed.
(extra white spaces in if statements look nice for code readability and you may consider
adjusting pylint settings).

**Comment by vkuznet** [comment]
Yes we need to include this to deal with CherryPy + DBS.

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
Ok, the functionality seems to be Ok since I didn't see anything falling apart. We shall see how it really behaves in cmsweb :) Thanks Valentin

**Pull request merged**